### ISSUE TITLE
[FVT]2.9.2:aix720 install error message "xcatd: possible BUG encountered by xCAT UDP service"

### ISSUE THREAD

**Issue by junxiawang** [xcat2/xcat-core#573]
How to reproduce:
1.install xcat on aix 720
During install
```
xcatd: possible BUG encountered by xCAT UDP service: Undefined subroutine &main::update_udpcontext_from_sslctl called at /opt/xcat/sbin/xcatd line 716.

xcatd: possible BUG encountered by xCAT UDP service: Undefined subroutine &main::update_udpcontext_from_sslctl called at /opt/xcat/sbin/xcatd line 716.

xcatd: possible BUG encountered by xCAT UDP service: Undefined subroutine &main::update_udpcontext_from_sslctl called at /opt/xcat/sbin/xcatd line 716.

xcatd: possible BUG encountered by xCAT UDP service: Undefined subroutine &main::update_udpcontext_from_sslctl called at /opt/xcat/sbin/xcatd line 716.

xcatd: possible BUG encountered by xCAT UDP service: Undefined subroutine &main::update_udpcontext_from_sslctl called at /opt/xcat/sbin/xcatd line 716.

xcatd: possible BUG encountered by xCAT UDP service: Undefined subroutine &main::update_udpcontext_from_sslctl called at /opt/xcat/sbin/xcatd line 716.

xcatd: possible BUG encountered by xCAT UDP service: Undefined subroutine &main::update_udpcontext_from_sslctl called at /opt/xcat/sbin/xcatd line 716.

xcatd: possible BUG encountered by xCAT UDP service: Undefined subroutine &main::update_udpcontext_from_sslctl called at /opt/xcat/sbin/xcatd line 716.

xcatd: possible BUG encountered by xCAT UDP service: Undefined subroutine &main::update_udpcontext_from_sslctl called at /opt/xcat/sbin/xcatd line 716.

xcatd: possible BUG encountered by xCAT UDP service: Undefined subroutine &main::update_udpcontext_from_sslctl called at /opt/xcat/sbin/xcatd line 716.

xcatd: possible BUG encountered by xCAT UDP service: Undefined subroutine &main::update_udpcontext_from_sslctl called at /opt/xcat/sbin/xcatd line 716.

xcatd: possible BUG encountered by xCAT UDP service: Undefined subroutine &main::update_udpcontext_from_sslctl called at /opt/xcat/sbin/xcatd line 716.

xcatd: possible BUG encountered by xCAT UDP service: Undefined subroutine &main::update_udpcontext_from_sslctl called at /opt/xcat/sbin/xcatd line 716.

xcatd: possible BUG encountered by xCAT UDP service: Undefined subroutine &main::update_udpcontext_from_sslctl called at /opt/xcat/sbin/xcatd line 716.

xcatd: possible BUG encountered by xCAT UDP service: Undefined subroutine &main::update_udpcontext_from_sslctl called at /opt/xcat/sbin/xcatd line 716.

xcatd: possible BUG encountered by xCAT UDP service: Undefined subroutine &main::update_udpcontext_from_sslctl called at /opt/xcat/sbin/xcatd line 716.

xcatd: possible BUG encountered by xCAT UDP service: Undefined subroutine &main::update_udpcontext_from_sslctl called at /opt/xcat/sbin/xcatd line 716.

xcatd: possible BUG encountered by xCAT UDP service: Undefined subroutine &main::update_udpcontext_from_sslctl called at /opt/xcat/sbin/xcatd line 716.
```

**Comment by hu-weihua** [comment]
Fix this defect for xcat2.9 by pull request #575

**Comment by daniceexi** [comment]
Following is the patch for this issue. You can just apply this patch before getting the fix in 2.9.3

~~~~
--- xcatd.orgin 2016-01-20 23:44:26.796000000 -0500
+++ xcatd.fix   2016-01-20 23:41:46.013000000 -0500
@@ -713,7 +713,8 @@
           $part = $socket->recv($data,1500);
            $packets{$part} = [$part,$data];
            } elsif ($hdl == $sslctl) {
-               update_udpcontext_from_sslctl(udpcontext=>$udpcontext,select=>$select);
+               next;
+               #update_udpcontext_from_sslctl(udpcontext=>$udpcontext,select=>$select);
            } elsif ($hdl == $discoctl) {  #got a discovery response....
            } else {
                print "Something is wrong in udp process (search xcatd for this string)\n";
@@ -933,7 +934,7 @@
       kill 2, $_;
    }
    if ($pid_UDP) {
-      kill 2, $pid_UDP;
+      kill 12, $pid_UDP;
    }
    if ($pid_MON) {
       kill 2, $pid_MON;
~~~~

**Comment by junxiawang** [comment]
test it

**Comment by whowutwut** [comment]
The problem was already fixed in xCAT 2.11 onwards.   Providing the pull requests for the other back level branches for completeness 

Fix for 2.9 branch, pull request #575 
Fix for 2.10 branch, pull request #738
